modules/virt-creating-data-volumes-using-storage-api.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-data-volumes.adoc
+
+[id="virt-creating-data-volumes-using-storage-api_{context}"]
+
+= Creating data volumes using the Storage API
+
+When you create a data volume using the *storage* API, the CDI optimizes your allocation based on the attributes of the mounted volume. You only have to specify the data volume name, namespace, and the amount of storage you want to allocate.
+
+For example:
+
+* When using Ceph RBD, your access mode is automatically set to `ReadWriteMany`, which improves efficiency during Live Migration, among other performance benefits.
+* File system overhead is automatically calculated for you. If you request two gigabytes of storage, you get exactly two gigabytes of storage, as opposed to a lesser amount when you use the *pvc* API.
```

Comment:
The PVC may be larger than 2 GB to ensure that there is at least 2GB of usable space to contain the VM disk.

Reply:
Changed to:

_File system overhead is automatically calculated for you. You will receive at least two gigabytes of usable space to contain the virtual machine disk._